_posts/2016-10-03-sr2017-kickstart-dates-announced.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Annoyingly this page name means that this article isn't available on the homepage for review yet. Quite neat that it does that though (and useful).

@jimmythompson Is there a way to fake the date so we can easily see such items?

For clarity: I think the name/date as is is fine -- it just make reviewing it a bit more complex.